core/foundation/inc/ROOT/TypeTraits.hxx

```diff
@@ -63,6 +64,11 @@ struct IsContainer {
    static constexpr bool value = Test<Test_t>(nullptr);
 };
 
+template<typename T>
+struct IsContainer<std::array_view<T>> {
+   static constexpr bool value = true;
+};
```

Comment:
Why is this here and not in `ROOT/RArrayView.hxx`? I don't understand.

Reply:
I think because we would like to have the TypeTraits to be aware of the array_view and not the opposite.

Reply:
Ah, sorry, somehow I though that this was a TDF header... makes sense.